test/integration/clientset_test.go

```diff
@@ -287,12 +287,12 @@ func testBrokerClient(sType server.StorageType, client servicecatalogclient.Inte
 		Name:      "test-name",
 	}
 
-	brokerServer.Spec.AuthInfo = &v1alpha1.BrokerAuthInfo{BasicAuthSecret: authSecret}
+	brokerServer.Spec.AuthInfo = &v1alpha1.BrokerAuthInfo{SecretRef: authSecret}
 
 	brokerUpdated, err := brokerClient.Update(brokerServer)
 	if nil != err ||
-		"test-namespace" != brokerUpdated.Spec.AuthInfo.BasicAuthSecret.Namespace ||
-		"test-name" != brokerUpdated.Spec.AuthInfo.BasicAuthSecret.Name {
+		"test-namespace" != brokerUpdated.Spec.AuthInfo.SecretRef.Namespace ||
```

Comment:
Well, those inline strings were all around the place before I added my test, just trying to be consistent.
If you insist on moving all of them into constants, I will :)